storage/innobase/dict/dict0dd.cc

```diff
@@ -1299,36 +1316,72 @@ int dd_table_open_on_dd_obj(dd::cache::Dictionary_client *client,
   return (1);
 }
 
-int dd_table_load_on_dd_obj(dd::cache::Dictionary_client *client,
-                            space_id_t space_id, const dd::Table &dd_table,
-                            dict_table_t *&table, THD *thd,
-                            const dd::String_type *schema_name, bool implicit) {
-  uint64 table_id = dd_table.se_private_id();
+std::tuple<int, std::vector<dict_table_t *>> dd_table_load_on_dd_obj(
```

Comment:
No marce, Rollback path asks for single table_id in space_id.
For export and stats, for one space_id, we load all tables in space_id. Then for each table object, we do export or stats.

Since we return multiple tables, we need a vector.

Reply:
@satya-bodapati got that we need a vector of dict_table_t, but what I mean is the return type bringing two information. 
I would rather have the function to return db_err and the vector be a in/out parameter

Reply:
Something like:

```
dberr_t dd_table_load_on_dd_obj(dd::cache::Dictionary_client *client,
                                      space_id_t space_id,
                                      const dd::Table &dd_table,
                                      table_id_t table_id, THD *thd,
                                      const dd::String_type *schema_name,
                                      bool implicit,
                                      std::vector<dict_table_t *> *table_vec);
```